### ISSUE TITLE
Support RPG Maker 2003 v1.11

### ISSUE THREAD

**Issue by Ghabry** [EasyRPG/Player#587]
This is a meta-bug to track all new features in RPG2k3 v1.11 and to discuss which are not worth implementing.

Changelog: https://docs.google.com/document/d/1sXlXZIXdjxVq5gh_lPFuFLMbMRzZe4BKPpRI13ZD2Qc/edit?pli=1
- [x] Transparent Window also affects Title Screen
- [X] When Battle Mode is one (traditional) the transparency setting of on-map message boxes is still respected (before they appeared opaque) (**->0.6.2**)
- [x] Support for skipping title screen. "In TestPlay Mode the state of Show/HideTitle button in the toolbar affects the behaviour: ShowTitle: Starts New Game directly, HideTitle: Go to Load Scene directly"
- [x] The “Y” key has now the function of an additional confirm button (such as Space, Enter and “Z”)
- [x] Picture operations (Show, Move, Erase) are not blocked by messages anymore. Do we already implement this?

Improved screen display: F5 can be used to open the video options. And a lot of different zoom options. Very low priority for me.

Event Commands (needs liblcf update):
- [x] Open Load Menu
- [x] Exit Game
- [x] Toggle Fullscreen
- [x] "Toggle ATB Wait Mode" <- Is Implemented but we our battle system is always ATB Wait
- [x] "Open Video Options" <- Just shows "OpenVideoOptions: Command not supported"

Conditional branch type “Other”
- [x] Savestate is available
- [x] Playtest mode is active <- questionable because of Anti-Cheat features
- [x] ATB wait mode <- Check is implemented
- [x] Fullscreen mode is active <- Nonsense on systems that have no window manager

An unsolved problem that is out of scope is, that v1.11 is a western europe only release, because other glyphs don't work which excludes the Asian countries where RPG2k/3 is still popular. I don't know if combining the editor with EasyRPG Player resolves font issues?

**Comment by Ghabry** [comment]
show_title is 0x6F in the System chunk of the LDB.
0 = Don't show, 1 = Show (Default)

```xml
<!-- Save state available? -->
<EventCommand>
	<code>12010</code>
	<indent>0</indent>
	<string></string>
	<parameters>11 0 0 0 0 0</parameters>
</EventCommand>
<!-- Play Test active? -->
<EventCommand>
	<code>12010</code>
	<indent>0</indent>
	<string></string>
	<parameters>11 1 0 0 0 0</parameters>
</EventCommand>
<!-- ATB is wait? -->
<EventCommand>
	<code>12010</code>
	<indent>0</indent>
	<string></string>
	<parameters>11 2 0 0 0 0</parameters>
</EventCommand>
<!-- Full screen active? -->
<EventCommand>
	<code>12010</code>
	<indent>0</indent>
	<string></string>
	<parameters>11 3 0 0 0 0</parameters>
</EventCommand>
```

**Comment by BlisterBoy** [comment]
I've downloaded the update, and it's awesome xp. Btw they have no more optimization issues, constant 60 fps !

**Comment by Ghabry** [comment]
I don't think the internals changed because of lack of source code. The changed part is the Blit to the screen, I guess RPG_RT renders in the old, buggy way and then new code copies it to the VRAM using better supported techniques.

**Comment by fmatthew5876** [comment]
> When Battle Mode is one (traditional) the transparency setting of on-map message boxes is still respected (before they appeared opaque)

This is fixed by #2078 so this issue should be finished when that goes in.
